src/less/rounded-off/base/rounded-off.less

```diff
@@ -53,6 +53,14 @@ button.expand-btn {
     --pagination-item-current-border-radius: @border-radius-none;
 }
 
+.progress-spinner {
+    --progress-spinner-image-url: url("https://ir.ebaystatic.com/cr/v/c1/skin/svg/spinner/v1/ds4/spinner.svg");
```

Comment:
These entries should be using a variable. Then the ds4/ds6 dist files will resolve correctly with ds4/ds6 paths. You'll probably need to create a new variable for the rounded case. DS4 doesn't have rounded or different asset for large, so DS4 css should always resolve to the old v1 spinner svg.

Reply:
hmm I guess I don't quite understand the need for a variable;
if they're turning the rounded off, then ds4/ds6 both large and small will all resolve to this path. It's the same spinner in ds4/ds6 for the old one.

Reply:
Ah, you're right. Ignore my comment! :-)